src/main/java/net/phptravels/gui/login/HomePage.java

```diff
@@ -17,84 +15,216 @@
 public class HomePage extends AbstractPage {
     private static final Logger LOGGER = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
 
-    @FindBy(xpath = "//div/nav/ul")
+    @FindBy(xpath = "//header[@class ='header-area']")
     private Topbar topbar;
     
-    @FindBy(id = "select2-tours_city-container")
-    private ExtendedWebElement destinationCombobox;
+    @FindBy(xpath ="//div//li[@data-position =1]")
+    private ExtendedWebElement hotelButton;
```

Comment:
I suggest to search this buttons not by position on the screen, but by name. Because if position will be changed locator will not work
<img width="599" alt="image" src="https://user-images.githubusercontent.com/103670717/163992178-47739e52-44cd-4a89-873f-c82757899adc.png">